### PR TITLE
add appcache audit & rename template

### PR DESCRIPTION
**Pull request by kaycebasques**
@ebidel would you plz review the app cache doc?

### CODE REVIEW

File: src/content/en/tools/lighthouse/audits/app-cache.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+description: Reference documentation for the "Site Does Not Use Application Cache" Lighthouse audit.
+
+{# wf_updated_on: 2017-01-04 #}
+{# wf_published_on: 2017-01-04 #}
+
+# Site Does Not Use Application Cache  {: .page-title }
+
+## Why the audit is important {: #why }
+
+Application Cache, also known as App Cache, is [deprecated][deprecated].
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/audits/app-cache.md]: AppCache is one word, camel case. Change throughout.

**Comment by kaycebasques** [comment]
Thanks @jpmedley, fixed

**Comment by jpmedley** [comment]
@ebidel This is short. Can you please look at this real quick. Thanks.

**Comment by nayphyoaung** [comment]
Thank you. 😁

**Comment by kaycebasques** [comment]
This is pretty short and uncontroversial so I'm just going to merge.

**Pull request merged**